test/helper.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+/**
+ * Removes the line break in the given file contents.
+ */
+exports.removeLineBreak = function (contents) {
+    return contents.toString().replace(/\r?\n|\r/g, '');
+};
```

Comment:
Sorry for confusing introduction of this helper. I will remove this.

The problem here is maybe that the expected value of `withpartial.ejs` is not 100% correct. So I'll try to fix that and remove this helper.

By the way, removing of line breaks starts at this commit (https://github.com/rogeriopvl/gulp-ejs/commit/8ff732f734087fca4e4fd95bb7eaf70bdd57d4ea). That was because of `ejs`'s breaking change and I think it's not a problem of this plugin.